client/reader/discover/discover-navigation.js

```diff
@@ -107,6 +108,15 @@ const DiscoverNavigation = ( { recommendedTags, selectedTab, width } ) => {
 					>
 						{ translate( 'Recommended' ) }
 					</SegmentedControl.Item>
+					{ config.isEnabled( 'reader/first-posts-stream' ) && (
```

Comment:
m guessing this may be due to wanting to internally test the stream results and quality before releasing to production? Or are there other reasons for the config check here?

Reply:
Yeah! I put it behind a flag so we can iterate a little bit before releasing to a broader audience. I think that would be a good idea since we had no input from design and we did big things, like removing the sidebar.